sqlness/examples/interceptor-replace/simple/replace.result

```diff
@@ -19,3 +19,26 @@ SELECT 1;
 
 03/14/2012, 01/01/2013 and 07/05/2014;
 
+-- SQLNESS TEMPLATE {"name": "test"}
```

Comment:
Reuse this old example to add test, also I plan to unite interceptor into one since their logic are the same.

Reply:
👍